frontend/src/metabase/meta/types/Field.js

```diff
@@ -1,10 +1,44 @@
 /* @flow */
```

Comment:
These type files could always have a text in the beginning explaining more or less thoroughly what the types are, are they one-to-one mapping to backend responses etc